examples/viewer_lib/ui/segmentation/segment_editor_ui.py

```diff
@@ -22,142 +25,185 @@
     SegmentationEffectThreshold,
 )
 
-from ..control_button import ControlButton
+from ..utils import AbstractToolUI, ControlButton, FlexContainer
 from .islands_effect_ui import IslandsEffectUI
 from .paint_effect_ui import PaintEffectUI
-from .segment_edit_dialog import SegmentEditDialog, SegmentEditDialogState
+from .segment_edit_ui import SegmentEditUI
 from .segment_list import SegmentList, SegmentListState
-from .segment_opacity_ui import SegmentationOpacityUI, SegmentOpacityState
+from .segment_rendering_ui import SegmentationRenderingUI, SegmentRenderingState
 from .threshold_effect_ui import ThresholdEffectUI
 
 
 @dataclass
 class SegmentEditorState:
     segment_list: SegmentListState = field(default_factory=SegmentListState)
-    segment_opacity: SegmentOpacityState = field(default_factory=SegmentOpacityState)
+    segment_rendering: SegmentRenderingState = field(default_factory=SegmentRenderingState)
```

Comment:
Already in other comments, but rename segment_rendering > segment_display and SegmentRenderingState > SegmentDisplayState